### PR TITLE
Config and CLI updates

### PR DESCRIPTION
**Pull request by winkerVSbecks**
* Config and CLI restructure
  * Split CLI and config docs. New pages: [CLI](https://deploy-preview-516--chromatic-docs.netlify.app/docs/cli/) and [Configure](https://deploy-preview-516--chromatic-docs.netlify.app/docs/configure/)
  * Using the new ConfigurationOptions component to documents all the options with filtering
  * Add info on config file schema
* Maintenance: 
  * Add code titles for markdown code snippets
  * Setup storybook, added stories for a few components
  * Refactored layout components
  * Upgrade astro
  * Update schema URL

### CODE REVIEW

File: src/components/Navigation/DropdownNav.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+import { within, userEvent } from "@storybook/test";
+import type { DropdownNavGroup } from "./DropdownNav";
+import { DropdownNav } from "./DropdownNav";
+
+const meta = {
+  title: "Components/DropdownNav",
+  component: DropdownNav,
+  parameters: {
+    viewport: {
+      defaultViewport: "mobile2",
```
Comment (jonniebigodes) [inline src/components/Navigation/DropdownNav.stories.tsx:11]: @winkerVSbecks I was wondering if you're ok with leaving out the `parameter` for now as it's introducing some inconsistent behavior with the other stories as it propagates it when navigating to other stories, which prevents the content from rendering correctly. When 8.3 lands and StoryGlobals are available, we can enable it for a specific set of stories, including this one, to test how it performs. Wouldn't you agree?

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@winkerVSbecks, thank you for putting together this pull request and improving the available configuration options. I left some items for you to look into when you can.

Let me know and we'll go from there.

**Comment by winkerVSbecks** [comment]
@jonniebigodes Thanks, those were very helpful suggestions. I made all the updates.

File: src/content/configuration/cli.mdx [hunk 2]
```diff
@@ -48,26 +62,49 @@ Optionally, you can install `chromatic` as a dependency while using the same scr
...
-- 👍 You won't need to install the package during local development if you're only running it in continuous integration.
-- 👎 It will be slower to run because the package has to be downloaded first.
+Chromatic CLI can be configured through options in `./chromatic.config.json` file (recommended) placed at the root of your project folder and/or by passing CLI flags.
 
-## Configuration
+<div class="aside">
+  Note: some options are only available as flags. For a full list of available
+  options, please refer to the [configuration options](/configuration-options)
+  page.
+</div>
```
Comment (jonniebigodes) [inline src/content/configuration/cli.mdx:73]: @winkerVSbecks, sorry for not catching this earlier, but it seems this URL is incorrect. Shouldn't it be `/docs/configure` instead?

File: src/components/Navigation/DropdownNav.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,116 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { within, userEvent } from "@storybook/test";
+import type { DropdownNavGroup } from "./DropdownNav";
+import { DropdownNav } from "./DropdownNav";
+
+const meta = {
```
Comment (jonniebigodes) [inline src/components/Navigation/DropdownNav.stories.tsx:6]: @winkerVSbecks, one item here, as you introduced Chromatic's modes feature, it now looks like we introduced a regression. Checking the latest build in Chromatic and navigating to this [component's story](https://66a7c58b7ecbd7bacf6a2264-znhbdnnbcj.chromatic.com/?path=/story/components-dropdownnav--collapsed) there's an error with the interaction test.

**Review by jonniebigodes — APPROVED** [review]

**Comment by jonniebigodes** [comment]
Thanks for addressing the feedback so promptly. Appreciate it 🙏 ! Just a couple of items (one more pressing than the other), and this is good to go. I've already approved it to unblock you.

File: src/components/Navigation/DropdownNav.stories.tsx (hunk 1 above)
Comment (winkerVSbecks) [reply src/components/Navigation/DropdownNav.stories.tsx]: yea, that's cause the nav is only visible on mobile. I'm going to have to add viewport back in.

Comment (winkerVSbecks) [reply src/components/Navigation/DropdownNav.stories.tsx]: Actually found a solution for it. Just needed to set a global default.

**Pull request merged**